sftp.c

```diff
@@ -810,6 +878,10 @@ do_ls_dir(struct sftp_conn *conn, const char *path,
 		m += strlen(tmp);
 		free(tmp);
 
+#ifdef IRONCORE
+		m += 2;  /*  Add space for lock icon or two spaces if not required  */
```

Comment:
Suggestion: Consider defining constants for this and the `"  "` not-locked string in the same area as the locked string.